scripts/manage-ruleset.sh

```
@@ -0,0 +1,295 @@
+function check_ruleset() {
+    local ruleset_id=$(get_ruleset_id)
+    
+    if [ -z "$ruleset_id" ]; then
+        echo -e "${YELLOW}⚠️  No ruleset named '$RULESET_NAME' found${NC}"
+        return 1
+    fi
+    
+    echo -e "${GREEN}✅ Found ruleset '$RULESET_NAME' (ID: $ruleset_id)${NC}"
+    
+    # Get ruleset details
+    local enforcement=$(gh api "repos/$REPO/rulesets/$ruleset_id" --jq '.enforcement')
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)